fsharp-backend/tests/Tests/Traces.Tests.fs

```diff
@@ -243,7 +256,7 @@ let testFunctionTracesAreStored =
 
 let tests =
   testList
-    "Analysis"
+    "Analysis" // should this be "traces"?
```

Comment:
I still myself often wondering where "analysis" starts and ends, and where other names are more appropriate.

Reply:
Good questions. Analysis properly refers to running the code on the client. It relies on traces, and so often we call the Trace stuff Analysis which makes sense in some but not all cases. Saving a Trace is not analysis, but fetching a Trace to analyze it might be.

Happy to explore the wording here (Trace is actually a new name in the new version, in the OCaml version they were called StoredEvents, StoredFunctionResults and StoredFunctionArguments.

Reply:
Thanks. let's call it OK for now (I'm removing the inline question) and follow up on it after I digest for a bit longer.